build.sbt

```diff
@@ -77,9 +76,22 @@ val core = crossProject(JVMPlatform, NativePlatform).crossType(CrossType.Pure).s
       }
       .toMap
   },
-
 ).nativeSettings(mimaPreviousArtifacts := Set.empty)
 
+val cli = crossProject(JVMPlatform)
+  .crossType(CrossType.Pure)
+  .settings(
+    name := "mima-cli",
+    crossScalaVersions ++= Seq(scala3),
```

Comment:
actually, was the intention here to set `crossScalaVersions := Seq(scala3)`? because with `++=` we get

```
sbt:mima> show cliJVM/crossScalaVersions
[info] * 2.12.20
[info] * 3.3.6
```

Reply:
yeah, oversight. I'll come back with a fix